client/src/pages/flows/FlowSideBar.tsx

```diff
@@ -157,43 +159,75 @@ interface FlowSideBarProps {
 const FlowsSideBar: React.FC<FlowSideBarProps> = ({ flows }) => {
   const navigate = useNavigate();
   const { flowId } = useParams();
+  const [filteredFlows, setFilteredFlows] = React.useState<FlowType[]>([]);
+
+  React.useEffect(() => {
+    setFilteredFlows(flows);
+  }, [flows]);
+
+  const handleSearch = (searchTerm: string) => {
```

Comment:
yes BUT: 

In this case, we already getting all the data as props to the componenet.
So do we need to "delay" the response to the user in the UI?  Dont you think that it will be better UX without the delay ?

Reply:
Agreed, no need to change that.